README.md

```diff
@@ -161,7 +161,7 @@ gcloud functions deploy new-dataset-function \
 --entry-point=new_dataset \
 --trigger-event-filters="type=google.cloud.storage.object.v1.finalized" \
 --trigger-event-filters="bucket=$PROJECT_NAME-europe-west2-dataset" \
---set-env-vars="DATASET_BUCKET_NAME=$PROJECT_NAME-europe-west2-dataset,SCHEMA_BUCKET_NAME=$PROJECT_NAME-europe-west2-schema,CONF=cloud-build,AUTODELETE_DATASET_BUCKET_FILE=True,RETAIN_DATASET_FIRESTORE=True,LOG_LEVEL=DEBUG,PROJECT_ID=$PROJECT_NAME,PUBLISH_SCHEMA_TOPIC_ID=ons-sds-publish-schema,PUBLISH_DATASET_TOPIC_ID=ons-sds-publish-dataset"
+--set-env-vars="DATASET_BUCKET_NAME=$PROJECT_NAME-europe-west2-dataset,SCHEMA_BUCKET_NAME=$PROJECT_NAME-europe-west2-schema,CONF=cloud-build,AUTODELETE_DATASET_BUCKET_FILE=True,RETAIN_DATASET_FIRESTORE=True,LOG_LEVEL=DEBUG,PROJECT_ID=$PROJECT_NAME,PUBLISH_SCHEMA_TOPIC_ID=ons-sds-publish-schema,PUBLISH_DATASET_TOPIC_ID=ons-sds-publish-dataset,SURVEY_MAP_URL=$SURVEY_MAP_URL"
```

Comment:
We'll need an actual path here instead of a variable

Reply:
@IsaiahLai111 Good point, just realised that we don't need this URL for the cloud function, just deleted this.